forms-flow-web/src/components/Dashboard/Dashboard.js

```diff
@@ -127,19 +126,17 @@ const Dashboard = React.memo(() => {
   useEffect(() => {
     const fromDate = getFormattedDate(dateRange[0]);
     const toDate = getFormattedDate(dateRange[1]);
-   // dispatch(setMetricsDateRangeLoading(true));
+    dispatch(setMetricsDateRangeLoading(true));
     setShowClearButton(searchText);
-    dispatch(
-      /*eslint max-len: ["error", { "code": 170 }]*/
-      fetchMetricsSubmissionCount(fromDate, toDate, searchBy, searchText, activePage, limit, sortsBy, sortOrder, (err, data) => {
-      })
-    );
-  }, [dispatch, searchBy, dateRange, searchInputBox, searchText, activePage, limit, sortsBy, sortOrder]);
+    /*eslint max-len: ["error", { "code": 170 }]*/
+    dispatch(fetchMetricsSubmissionCount(fromDate, toDate, searchBy, searchText, activePage, limit, sortsBy, sortOrder, (err, data) => { }));
```

Comment:
run prettier extension which will format the code automatically, refer contribution guidelines. `shift+alt+f` to format